python_modules/dagster/dagster/_seven/compat/pendulum.py

```diff
@@ -60,6 +60,9 @@ def create_pendulum_time(year, month, day, *args, **kwargs):
 )
 
 
+pendulum_time_travel = pendulum.travel_to if _IS_PENDULUM_3 else pendulum.time
```

Comment:
not to be too pedantic, but this is actually subtly different I believe.

`pendulum.travel_to` keeps the clock ticking, whereas `pendulum.time` freezes the time entirely. so think actually you'd want lambda t: pendulum.travel_to(t, freeze=true) here

Reply:
with that in mind, I think this should be called pendulum_freeze_time, as I don't think the older versions actually support the non-frozen behavior